pkg/kubelet/apis/deviceplugin/v1alpha/api.proto

```diff
@@ -96,6 +96,10 @@ message AllocateResponse {
 	repeated Mount mounts = 2;
 	// Devices for the container.
 	repeated DeviceSpec devices = 3;
+	// Annotations for the container to pass to the runtime.
+	// These annotations are generated by other components (i.e.,
+	// not users). Currently, only device plugins populate the annotations.
```

Comment:
This comment makes sense in `pkg/kubelet/container/runtime.go` but needs to be modified here.